eng/pipelines/common/ui-tests.yml

```diff
@@ -209,6 +209,44 @@ stages:
                       testConfigurationArgs: "CollectionView2"
                       skipProvisioning: ${{ parameters.skipProvisioning }}
 
+  - stage: ios_ui_tests_mono_carv2
+    displayName: iOS UITests Mono CarouselView2
+    dependsOn: build_ui_tests
+    jobs:
+      - ${{ each project in parameters.projects }}:
+        - ${{ if ne(project.ios, '') }}:
+          - ${{ each version in parameters.iosVersions }}:
+            - ${{ if not(containsValue(project.iosVersionsExclude, version)) }}:
+              - job: CARV2_ios_ui_tests_mono_${{ project.name }}_${{ replace(version, '.', '_') }}
+                timeoutInMinutes: 240 # how long to run the job before automatically cancelling
```

Comment:
Maybe 240 minutes (4 hours) seems quite long, but my point here is that we are repeating the value in several places. Could include a variable and reuse it everywhere?

Reply:
Reduced to 2 hours and moved to a variable